core/src/test/resources/rewrite/RedundantBraces.stat

```diff
@@ -159,3 +160,83 @@ object a {
     } // end
   }
 }
+<<< redundant enclosing braces in str interpolation
+object a {
+  s"Hello ${value}"
+}
+>>>
+object a {
+  s"Hello $value"
+}
+<<< not redundant enclosing braces in str interpolation
+object a {
+  s"Hello ${foo}bar"
+}
+>>>
+object a {
+  s"Hello ${foo}bar"
+}
+<<< enclosing braces in str interpolation
+object a {
+  s"Hello ${d.toString}"
+}
+>>>
+object a {
+  s"Hello ${d.toString}"
+}
+<<< empty str interpolation
+object a {
+  s""
+}
+>>>
+object a {
+  s""
+}
+<<< no braces in str interpolation
+object c {
+  s"${a}"
+}
+>>>
+object c {
+  s"$a"
+}
+<<< two args in str interpolation
+object c {
+  s"${a}${b}"
+}
+>>>
+object c {
+  s"$a$b"
+}
+<<< spaces before and after arg in str interpolation
+object c {
+  s" ${a} "
+}
+>>>
+object c {
+  s" ${a} "
+}
+<<< special symbol after arg no braces in str interpolation
+object c {
+  s"${a}-"
+}
+>>>
+object c {
+  s"${a}-"
```

Comment:
i was thinking about check like `value.isEmpty` or starts with \n, \r or anything like this! what else should it be? minus/plus... is there kind a list of them? or do you think there is a better approach?